### DISCUSSION TITLE
[Art competition] Information Security Technology

### DISCUSSION THREAD

**Discussion by Leejinjae-developer** [prosyslab-classroom/cs348-information-security#416] [Art competition]
** Upload your picture and provide the following description. Delete this line. **

## Short Description
For me, [Information Security Technology] means [digital art, many lotus flowers growing competitively high in dirty pond, colorful and beautiful petals, dirty and polluted pond, a bright sky, a dark pond]".

## Long Description (up to 3 sentences)
The evil instinct of humans who do not hesitate to hack into other people's information for their own benefit resembles a dirty pond.
The sophistication and beauty of safe and stable information security technology, which is constantly developed based on wickedness as a driving force, resembles that of lotus flowers.
Information security technology, which is constantly evolving without complacency to defend against attackers, resembles a lotus flower that grows higher in competition with surrounding objects.

![DALL·E 2023-05-31 11 48 04 - digital art, many lotus flowers growing competitively high in dirty pond, colorful and beautiful petals, dirty and polluted pond, a bright sky, a dark](https://github.com/prosyslab-classroom/cs348-information-security/assets/128210419/4568d793-dfa6-458d-8170-620e316f6ad2)

**Comment by KihongHeo** [comment]
Good. Your information security tech looks so gloomy.
